packages/core/parcel/package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
-  "bin": "src/bin.js",
+  "bin": "lib/bin.js",
```

Comment:
Revert this (artifact from `yarn build`)

Reply:
Fixed. How do you guys avoid this happening all the time?